main.py

```diff
@@ -108,6 +163,69 @@ def toast():
 toast()
 """
 
+    def storage4kivy_open(self):
+        '''
+        Calls Android Chooser to pick a file and then chooser_callback
+        '''
+        if platform == "android":
+            Chooser(self.chooser_callback).choose_content()
+
+    def chooser_callback(self,uri_list):
+        reading = ""
+        '''
+        Using chooser from androidstorage4kivy to read file content.
+        Limits : Can retrieve only txt file, sure there is a way to
+        retrieve others extension files
+        '''
+        try:
+            ss = SharedStorage()
+            for uri in uri_list: # only one list
+                
+                # copy to private cache
+                path = ss.copy_from_shared(uri)
+
+                #read content from cache
+                try:
+                    with open(path,'r',encoding = 'utf-8') as fread:
+                        reading = fread.read()
+
+                    #Clock is needed otherwise throw thread error
+                    Clock.schedule_once(lambda dt: self.populate_label(reading))
+
+                except Exception as e:
+                    toast(f"1.cannot open file path : {str(path)}  because : {str(e)}")
+
+        except Exception as e:
+            toast("something wrong with package androidstorage4kivy" + str(e))
+
+    def populate_label(self,reading):
+
+        if self.laoding_without_adding:
+            self.root.ids.code.text = reading
+        else:
+            self.root.ids.code.text += "\n" + reading
+
+    def storage4kivy_save(self):
```

Comment:
@javixav instead of using the name `storage4kivy_save`, why not use `save_file`. This app might also be deployed on a desktop and the name `storage4kivy_save` can cause confusion in the future